### ISSUE TITLE
inDispute

### ISSUE THREAD

**Issue by kusala9** [iho-ohi/S-57-to-S-101-conversion-sub-WG#11]
Recent discussions have raised some questions over how "inDispute" should be converted. These are mainly when the key terms (in the INFORM) are attached to a CTNARE, overlapping part of a TESARE or EXEZNE.

The current guidance would require cutting the underlying EXEZNE/TESARE into pieces, to ensure the inDispute attribute only extends to the original CTNARE, not to the entire EXEZNE/TESARE.

An alternative approach would require the originating data producer pre-partitioning the S-57 features to ensure the CTNARE is only coincident with one of the pieces - this piece would then be assigned the inDispute attribute. 

Both methods have merit (generally, it seems unacceptable the entire EXEZNE/TESARE is attributed with inDispute if only a certain portion of it is covered by the CTNARE) but some words recommending a particular course of action or description of the options would be good for the document....

**Comment by tdepuyt** [comment]
@Christian-Shom 

[Multiple ADMARE.zip](https://github.com/iho-ohi/S-57-to-S-101-conversion-sub-WG/files/7157985/Multiple.ADMARE.zip)
[Multiple CTNARE over FSHZNE.zip](https://github.com/iho-ohi/S-57-to-S-101-conversion-sub-WG/files/7157986/Multiple.CTNARE.over.FSHZNE.zip)

**Comment by JeffWootton** [comment]
The inDispute attribute should definitely not be populated for the entire area (including the area not in dispute).  My preference would be, if a process can be implemented for dealing with the change in the features constituting the Skin of the Earth between S-57 and S-101, to implement a similar process and have the S-57 administrative area feature "cookie cut" along the edge of the associated CTNARE feature and populate the inDispute attribute for the resultant administrative area coincident with the CTNARE as _True_.
The remaining issue is whether the CTNARE should be retained through to S-101.  This is dependent I think on answering questions related to whether the INFORM and/or TXTDSC from the CTNARE is converted to the information complex on the relevant administrative area feature in S-101; and on whether there will be a portrayal impact in S-100 ECDIS for the population of inDispute as _True_.

**Comment by kusala9** [comment]
some suggested wording below. Obviously if the data producer pre-partitions the TESARE and covers the in dispute section with a CTNARE then the conversion is easy. If they don't pre-partition it then the suggested wording below means the indispute attribution is restricted only to the (created) section covered by the CTNARE. I don't believe this will cause any issues although it does require geometry creation - this should not present any problems for converters as geometry creation is required by a number of other processes....


> The following additional requirements for S-57 attribution must be noted:
> •	The S-101 attribute in dispute has been introduced in S-101 to provide an indication to the mariner that a territorial sea area is in dispute.  This information may be encoded in S-57 using a CTNARE object covering the area that is in dispute.  In order for this information to be converted across to S-101, the text string encoded in INFORM on the CTNARE must be in the format In dispute.  The S-101 Caution Area feature resulting from the conversion of the CTNARE should be removed from the converted S-101 dataset in this case.


Suggested wording….

•	The S-101 attribute in dispute has been introduced in S-101 to provide an indication to the mariner that some or all of a territorial sea area is in dispute.  This information may be encoded in S-57 using a CTNARE object covering the area that is in dispute.  In order for this information to be converted across to S-101, the text string encoded in INFORM on the CTNARE must be in the format In dispute. If the CTNARE only covers a section of the TESARE then in this case the Territorial Sea Area created should be partitioned. One Territorial Sea Area should be created coincident with the CTNARE, with the in dispute attribute set to true and another Territorial Sea Area created with the remaining area, with the in dispute attribute set to false. The S-101 Caution Area feature resulting from the conversion of the CTNARE should be removed from the converted S-101 dataset in this case.

**Comment by Christian-Shom** [comment]
I think this is really the kind of thing that needs testing, based on actual cases. We probably have situations where 2 TESARE overlap and the CTNARE does not share the geometry of the overlap. In such cases, the "In dispute" area should be determined based on the overlap but not on the CTNARE...
Additionnally, the converters may be able not to convert the CTNARE in these cases (so that the Data Provider will not have to remove the S-101 Caution Area).
In the end, S-57 data preparation might be unavoidable in many cases.
I am of the opinion to go with the wording suggested by Jeff hereafter and see what the tests will show.
![image](https://user-images.githubusercontent.com/71139640/157830635-fec525ed-4f10-4f91-9144-22a7b3286ed6.png)

**Comment by TDYCARHugh** [comment]
I agree with Christian and the need for testing/considering actual cases. It is easy to make a rule based on the easy/simple cases but what about the not so simple cases.  I am not an authority on such territorial issues but I think there may be a few things we might need to consider:

1. This is potentially very sensitive topic and perhaps problematic to state that it 'must' or even 'should'  be done a specific way in every case.
2. Whether an agency is actually willing to acknowledge that their boundary is in dispute.
3. Whether an agency is willing to include ( and in some way acknowledge) an alternative disputed boundary geometry in their ENC. The alternate boundaries would be needed to form the actual portion in dispute.
4. Does the disputed portion form a simple polygon?  What if two disputed boundaries criss-cross and overlap? The result of a cutting operation could form several disjoint polygons (complex).
5. Would an agency want a single TESARE in S-57 to become multiple TerritorialSeaArea in S-101? Or would they want the same set of TESARE in both.  
6. The string 'In dispute' or 'Eastern regions in dispute' could be carried as INFORM on the TESARE.
7. If a CTNARE is warranted in S-57 ENC for a region in dispute then is it not also warranted in S-101?  Is a TerritorialSeaArea with attribute inDispute enough?  What if it is dangerous to be within that area such as you might be fired upon or have your vessel seized. There is no distinction in the boolean inDispute to identify how serious or dangerous the dispute might be.  In other words dropping a CTNARE if it overlaps the inDispute area may not be simple to decide.


Perhaps we should consider that it should be up to the producer to decide how they want this to be depicted in their ENC.  If they want separate TerritorialSeaArea areas in converted S-101 then make separate TESARE in S-57.  Use the INFORM on TESARE if it is just a note acknowledging in dispute or make a CTNARE if it is important for caution when navigating and retain that Caution area in the S-101 ENC.

**Comment by JeffWootton** [comment]
I am not aware of any negative feedback as a result of data conversion from S-57 to S-101 using the guidance that is currently in S-65 Annex B. Having said this, however, I am also not sure how many actual areas that are in dispute have been converted.

Suggest close this Issue - can be re-opened if any further feedback is provided.
